### DISCUSSION TITLE
建议作者同步再做一个适配另外火热的gptSOVITS的api或者再开一个项目，这类需求很多的

### DISCUSSION THREAD

**Discussion by anthonyshibo** [lenML/Speech-AI-Forge#38] [想法 | Ideas]
感谢作者，对于想平移国外项目的人来说太实用了

**Comment by zhzLuke96** [comment]
嗯，有计划实现 xttsv2 接口

**Reply by zhzLuke96** [reply]
已经实现 #42 

具体配置方法可以看 readme https://github.com/lenML/ChatTTS-Forge?tab=readme-ov-file#421-how-to-link-to-sillytavern
